src/components/TodoItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  tempTodo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ tempTodo }) => (
```

Comment:
use desctructure assignment
```suggestion
export const TodoItem: React.FC<Props> = ({ tempTodo: { title, .... } }) => (
```